paz/models/detection/model_util.py

```diff
@@ -0,0 +1,268 @@
+################################################
+### The functions in this file are direct    ###
+### copy of the ones present in the original ###
+### Frustum PointNet work                    ###
+################################################

```

Comment:
this should be removed. The name of what is done in the module should be self explanatory from the module name alone and the individual documentation of the functions

Reply:
Removed model_util.py